cmd/kpod/load.go

```diff
@@ -95,14 +95,19 @@ func loadCmd(c *cli.Context) error {
 		output = os.Stdout
 	}
 
+	options := libpod.CopyOptions{
+		SignaturePolicyPath: c.String("signature-policy"),
+		Writer:              output,
```

Comment:
Can we change the variable name output to writer to be consistent with the ones below?